recipe/meta.yaml

```diff
@@ -1,33 +1,31 @@
 {% set name = "nbclient" %}
-{% set version = "0.5.13" %}
-{% set build = 0 %}
+{% set version = "0.8.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 40c52c9b5e3c31faecaee69f202b3f53e38d7c1c563de0fadde9d7eda0fdafe8
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nbclient-{{ version }}.tar.gz
+  sha256: f9b179cd4b2d7bca965f900a2ebf0db4a12ebff2f36a711cb66861e4ae158e55
 
 build:
-  number: {{ build }}
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
     - jupyter-execute = nbclient.cli:main
 
 requirements:
   host:
     - python
     - pip
-    - setuptools
-    - wheel
```

Comment:
so if setuptools are not used then we do not need wheel as well ?

Reply:
hatchling has the required code to build wheels